addons/website_slides/views/slide_slide_views.xml

```diff
@@ -203,67 +203,52 @@
-                                                <i class="fa fa-file-code-o me-2" aria-label="article" role="img" title="Article"/>
-                                            </t>
-                                            <t t-elif="record.slide_category.raw_value == 'video'">
-                                                <i class="fa fa-file-video-o me-2" aria-label="Video" role="img" title="Video"/>
-                                            </t>
-                                            <t t-elif="record.slide_category.raw_value == 'quiz'">
-                                                <i class="fa fa-flag me-2" aria-label="Quiz" role="img" title="Quiz"/>
-                                            </t>
-                                            <t t-else=""><i class="fa fa-file-pdf-o me-2" aria-label="Document" role="img" title="Document"/></t>
-                                            <field name="slide_category"/>
-                                        </span>
-                                        <span>
-                                            <i class="fa fa-clock-o me-2" aria-label="Duration" role="img" title="Duration"/><field name="completion_time" widget="float_time"/>
-                                        </span>
-                                        <span>
-                                            <i class="fa fa-question me-2" aria-label="Number of Questions" role="img" title="Number of Questions"/><field name="questions_count"/>
-                                        </span>
-                                        <span>
-                                            <i class="fa fa-eye me-2" aria-label="Views" role="img" title="Views"/><field name="total_views"/>
-                                        </span>
-                                        <field name="user_id" widget="many2one_avatar_user"/>
+                        <t t-name="kanban-card" class="flex-row">
+                            <aside class="o_kanban_aside_full">
+                                <t t-if="record.image_128.raw_value">
+                                    <div class="o_kanban_image_fill position-relative w-100">
+                                        <field name="image_128" class="h-100" widget="image" options="{'img_class': 'object-fit-cover'}"/>
+                                        <field name="channel_id" class="o_website_slides_inner_image position-absolute bottom-0 end-0 bg-light" widget="image" options="{'preview_image': 'image_128', 'img_class': 'object-fit-contain'}"/>
                                     </div>
-                                </div>
-                            </div>
+                                </t>
+                                <t t-else="">
+                                    <img src="/website_slides/static/src/img/channel-training-default.jpg" class="w-100" options="{'img_class': 'object-fit-cover'}" t-att-alt="record.id.value"/>
```

Comment:
this alt doesn't really make sense does it? This is the fallback/default image which is static, so I would have expected a static string describing the image instead